docs/ibc/overview.md

```diff
@@ -10,131 +10,147 @@ Learn what IBC is, its components and use cases. {synopsis}
+chain. 
+
+The supported IBC clients are:
 
-* [Solo Machine light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/06-solomachine): devices such as phones, browsers, or laptops.
-* [Tendermint light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint): The default for SDK-based chains,
+* [Solo Machine light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/06-solomachine): Devices such as phones, browsers, or laptops.
+* [Tendermint light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint): The default for Cosmos SDK-based chains.
 * [Localhost (loopback) client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/09-localhost): Useful for
-testing, simulation and relaying packets to modules on the same application.
+testing, simulation, and relaying packets to modules on the same application.
 
 ### [Connections](https://github.com/cosmos/ibc-go/blob/main/modules/core/03-connection)
 
-Connections encapsulate two `ConnectionEnd` objects on two seperate blockchains. Each
-`ConnectionEnd` is associated with a client of the other blockchain (ie counterparty blockchain).
+Connections encapsulate two `ConnectionEnd` objects on two separate blockchains. Each
+`ConnectionEnd` is associated with a client of the other blockchain (for example, the counterparty blockchain).
 The connection handshake is responsible for verifying that the light clients on each chain are
 correct for their respective counterparties. Connections, once established, are responsible for
-facilitation all cross-chain verification of IBC state. A connection may be associated with any
+facilitation all cross-chain verification of IBC state. A connection can be associated with any
 number of channels.
 
 ### [Proofs](https://github.com/cosmos/ibc-go/blob/main/modules/core/23-commitment) and [Paths](https://github.com/cosmos/ibc-go/blob/main/modules/core/24-host)
   
 In IBC, blockchains do not directly pass messages to each other over the network. Instead, to
-communicate, a blockchain will commit some state to a specifically defined path reserved for a
-specific message type and a specific counterparty (perhaps storing a specific connectionEnd as part
-of a handshake, or a packet intended to be relayed to a module on the counterparty chain). A relayer
-process monitors for updates to these paths, and will relay messages, by submitting the data stored
-under the path along with a proof to the counterparty chain. The paths that all IBC implementations
-must use for committing IBC messages is defined in
-[ICS-24](https://github.com/cosmos/ics/tree/master/spec/core/ics-024-host-requirements) and the proof
-format that all implementations must be able to produce and verify is defined in this [ICS-23 implementation](https://github.com/confio/ics23).
+communicate, a blockchain commits some state to a specifically defined path that is reserved for a
```

Comment:
I applied the guideline to use present tense, per https://developers.google.com/style/tense